### PR TITLE
about/for-proposals: Add RSE section

### PR DESCRIPTION
**Pull request by rkdarst**
- We needed a RSE text which can be included in grant applications.
  This is the start of that.
- Review: this isn't very good yet, and I found it hard to make
  something very general that can be copied and pasted like the other
  text on this page.  Please give it a careful read and revise it as
  much as you can.

### CODE REVIEW

**Comment by rantahar** [comment]
I don't really have anything to add to this. Should we just merge?

File: about/for-proposals.rst [hunk 1]
```diff
@@ -54,6 +54,32 @@ access controlled by passwords and individual-level authorization, and
...
+--------------------------------------
+
+(this text must especially be tuned to your grant)
+
+This grant will make use of the Aalto Research Software Engineer
+program to hire high-quality TOPIC specialists.  This program provides
+PhD-level specialists to work on THINGS, which allows the other staff
+on this project to focus on YYY.  These staff do not need to be
+independently recruited, are available for consultation before and
+answer the project.  This service is provided by Aalto Scientific
```
Comment (wmvanvliet) [inline about/for-proposals.rst:68]:
```suggestion
after the project.  This service is provided by Aalto Scientific
```

**Comment by eglerean** [comment]
Please check if my edits are fine: I explicitly called our RSEs "research software engineers" but @rkdarst 's original intention was to not have it like that, you can change those bits before merging.

File: about/for-proposals.rst [hunk 2]
```diff
@@ -54,6 +54,32 @@ access controlled by passwords and individual-level authorization, and
...
+Computing, which also provides high-performance computing resources for our project.
+The Research Software Engineering service is integrated into computing
+services as a consistent package.
+
+(for basic service, for now only SCI) The service is available as a
+basic consulting service for free.
+
+(for paid services) This project receives dedicated service from the
+Research Software Engineering group, funded as researcher salary from
+this grant.  During this period, on of Aalto research software engineers joins this
```
Comment (rkdarst) [inline about/for-proposals.rst:78]:
```suggestion
this grant.  During this period, one of the Aalto research software engineers joins this
```

**Pull request merged**